src/main/java/homework_2/TrafficLight.java

```diff
@@ -0,0 +1,38 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TrafficLight {
```

Comment:
Good. Simple solution, but logical evaluations could be confusing

Reply:
Opt: to be fixed
![image](https://user-images.githubusercontent.com/58872509/126037097-dbab8e04-4cd1-4770-a6eb-27a299548e66.png)